core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -30,7 +30,7 @@
             {{ tab.text }}
           </div>
         </div>
-        <div class="oppia-contributions-option-group-label"
+        <div class="oppia-contributions-option-group-label e2e-test-accomplishments-tab-enabled-indicator"
```

Comment:
@lkbhitesh07 could you please explain why this change was required?

Reply:
Ah! Forgot to remove this part. I actually edited one of the e2e test file and this was required there but I reverted those changes so this change is not needed anymore.
I have removed it, thanks for catching.